examples/bert/yamls/main/mcloud_run_a100_40gb.yaml

```diff
@@ -17,13 +17,21 @@ image: mosaicml/pytorch:1.13.1_cu117-python3.10-ubuntu20.04
 integrations:
 - integration_type: git_repo
   git_repo: mosaicml/examples
-  git_branch: v0.0.3 # use your branch
+  git_branch: v0.0.4 # use your branch
   # git_commit: # OR use your commit hash
   pip_install: -e .[bert]
   ssh_clone: false # Should be true if using a private repo
+
+# We are fetching, converting, and training on the 'train_small' split
+# as it is small and quick to get going for this demo.
+# For real training runs, follow the instructions in `examples/bert/README.md`
+# to convert and host the full 'train' dataset.
 command: |
   cd examples/examples/bert
-  composer main.py /mnt/config/parameters.yaml
+  python ../common/convert_dataset.py --dataset c4 --data_subset en --out_root ./my-copy-c4 --splits train_small val
+  composer main.py /mnt/config/parameters.yaml \
+    train_loader.dataset.split=train_small \
+    data_local=./my-copy-c4
```

Comment:
```suggestion
  python ../common/convert_dataset.py --dataset c4 --data_subset en --out_root ./my-copy-c4 --splits train_small val_val
  composer main.py /mnt/config/parameters.yaml \
    train_loader.dataset.split=train_small \
    eval_loader.dataset.split=val_small \
    data_local=./my-copy-c4
```

Reply:
i should change to `val_small` everywhere? theres a bunch of yamls and readmes that all do `train_small` and `val`

Reply:
@abhi-mosaic got thoughts?

Reply:
I think either is ok, val_small does help a bit esp since it only gets 8-way parallelism, but it's also more annoying to have two fields to edit. Dont have a strong opinion

Reply:
ok im gonna stick with `val`